### ISSUE TITLE
Script issue with ATxmega32A4U

### ISSUE THREAD

**Issue by mcuee** [MCUdude/avrdude-test-script#9]
avrdude is okay but not the script
```
$ ./avrdude_test_script_avrispmk2_xmega32a4u.sh
Prepare "-cavrispmkii -B1 -patxmega32a4u" and press 'enter' or 'space' to continue. Press any other key to skip
✅ eesave fuse bit set, cleared and verified
avrdude OS error: file test_files/the_quick_brown_fox_36864B.hex is not readable: No such file or directory
avrdude OS error: file test_files/the_quick_brown_fox_36864B.hex is not readable: No such file or directory
avrdude OS error: unable to open test_files/the_quick_brown_fox_36864B.hex: No such file or directory
avrdude error: cannot determine file format for test_files/the_quick_brown_fox_36864B.hex, specify explicitly
avrdude error: read from file test_files/the_quick_brown_fox_36864B.hex failed
❌ the_quick_brown_fox_36864B.hex flash -U write/verify
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -Uflash:w:test_files/the_quick_brown_fox_36864B.hex -Uflash:v:test_files/the_quick_brown_fox_36864B.hex" failed
✅ the_quick_brown_fox_1024B.hex eeprom -U write/verify
avrdude OS error: file test_files/lorem_ipsum_36864B.srec is not readable: No such file or directory
avrdude OS error: file test_files/lorem_ipsum_36864B.srec is not readable: No such file or directory
avrdude OS error: unable to open test_files/lorem_ipsum_36864B.srec: No such file or directory
avrdude error: cannot determine file format for test_files/lorem_ipsum_36864B.srec, specify explicitly
avrdude error: read from file test_files/lorem_ipsum_36864B.srec failed
❌ lorem_ipsum_36864B.hex flash -U write/verify
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -Uflash:w:test_files/lorem_ipsum_36864B.srec -Uflash:v:test_files/lorem_ipsum_36864B.srec" failed
✅ lorem_ipsum_1024B.srec eeprom -U write/verify
✅ 0xff_1024B.hex eeprom -U write/verify
❌ the_quick_brown_fox_36864B.hex:a flash -T write/verify
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -T "write flash test_files/the_quick_brown_fox_36864B.hex:a"" failed
✅ the_quick_brown_fox_1024B.hex:a eeprom -T write/verify
avrdude error: (write) data test_files/lorem_ipsum_36864B.srec: unable to read the file: No such file or directory
❌ lorem_ipsum_36864B.srec flash -T write/verify
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -T "write flash test_files/lorem_ipsum_36864B.srec"" failed
✅ lorem_ipsum_1024B.srec eeprom -T write/verify
✅ cola-vending-machine.raw flash -T/-U write/verify
avrdude OS error: file test_files/holes_pack_my_box_36864B.hex is not readable: No such file or directory
avrdude OS error: unable to open test_files/holes_pack_my_box_36864B.hex: No such file or directory
avrdude error: cannot determine file format for test_files/holes_pack_my_box_36864B.hex, specify explicitly
avrdude error: read from file test_files/holes_pack_my_box_36864B.hex failed
❌ holes_pack_my_box_36864B.hex:a flash -U write
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -Uflash:w:test_files/holes_pack_my_box_36864B.hex:a" failed
❌ holes_the_five_boxing_wizards_36864B.hex flash -T write
➡️ command "./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -T "write flash test_files/holes_the_five_boxing_wizards_36864B.hex"" failed
✅ holes_pack_my_box_1024B.hex:a eeprom -U write
✅ holes_the_five_boxing_wizards_1024B.hex eeprom -T write
✅ random_data_256B.bin usersig -T/-U write/read

One or more avrdude "-cavrispmkii -B1 -patxmega32a4u" tests failed. Do you want to retry this particular test? (y/n): n


$ ./avrdude -qq -cavrispmkii -B1 -patxmega32a4u -t
avrdude> part
ATxmega32A4U with programming modes PDI, SPM

Memory           Size  Pg size     Offset
-----------------------------------------
eeprom           1024       32   0x8c0000
flash           36864      256   0x800000
application     32768      256   0x800000
apptable         4096      256   0x807000
boot             4096      256   0x808000
fuse0               1        1   0x8f0020
fuse1               1        1   0x8f0021
fuse2               1        1   0x8f0022
fuse4               1        1   0x8f0024
fuse5               1        1   0x8f0025
lock                1        1   0x8f0027
prodsig/sigrow     52        1   0x8e0200
signature           3        1  0x1000090
usersig           256      256   0x8e0400
io               4096        1          0
sram             4096        1     0x2000

Variants          Package  F max   T range         V range
-----------------------------------------------------------------
ATxmega32A4U-AN   TQFP44   32 MHz  [0 C,   105 C]  [1.6 V, 3.6 V]
ATxmega32A4U-ANR  TQFP44   32 MHz  [0 C,   105 C]  [1.6 V, 3.6 V]
ATxmega32A4U-AU   TQFP44   32 MHz  [-40 C,  85 C]  [1.6 V, 3.6 V]
ATxmega32A4U-AUR  TQFP44   32 MHz  [-40 C,  85 C]  [1.6 V, 3.6 V]
ATxmega32A4U-CU   BGA49    32 MHz  [-40 C,  85 C]  [1.6 V, 3.6 V]
ATxmega32A4U-CUR  VFBGA49  N/A     [N/A,     N/A]  [N/A,     N/A]
ATxmega32A4U-M7   VQFN44   32 MHz  [-40 C, 105 C]  [1.6 V, 3.6 V]
ATxmega32A4U-MH   QFN44    32 MHz  [-40 C,  85 C]  [1.6 V, 3.6 V]
ATxmega32A4U-MHR  VQFN44   32 MHz  [-40 C,  85 C]  [1.6 V, 3.6 V]
avrdude> quit
```

**Comment by MCUdude** [comment]
> avrdude OS error: file test_files/the_quick_brown_fox_36864B.hex is not readable: No such file or directory

I haven't generated files that are 36864 bytes large.

**Comment by MCUdude** [comment]
Resolved in 24fd6d0
